ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/InternalCustomerCenter.kt

```diff
@@ -38,11 +54,29 @@ import kotlinx.coroutines.launch
 @Composable
 internal fun InternalCustomerCenter(
     modifier: Modifier = Modifier,
+    onDismiss: () -> Unit,
     viewModel: CustomerCenterViewModel = getCustomerCenterViewModel(),
 ) {
     val state by viewModel.state.collectAsState()
     val coroutineScope = rememberCoroutineScope()
     val context = LocalContext.current
+
+    if (state is CustomerCenterState.NotLoaded) {
```

Comment:
This was the easiest to refresh the data when re-starting the Customer Center again. Otherwise if re-opening the Customer Center, the state is the same as before closing it.